pkgs/top-level/all-packages.nix

```diff
@@ -15513,6 +15521,13 @@ with pkgs;
     targetLlvm = targetPackages.llvmPackages_15.llvm or llvmPackages_15.llvm;
   }));
 
+  llvmPackages_16 = recurseIntoAttrs (callPackage ../development/compilers/llvm/16 ({
+    inherit (stdenvAdapters) overrideCC;
+    buildLlvmTools = buildPackages.llvmPackages_16.tools;
+    targetLlvmLibraries = targetPackages.llvmPackages_15.libraries or llvmPackages_16.libraries;
```

Comment:
Should this be 16, rather than 15?

Reply:
Apparently this is already fixed now -> marking as resolved. Thanks for this catch :)